GitUp/Application/Base.lproj/AuthenticationWindowController.xib

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we add constraints to this view as well? Lots of warnings on this page as it is, probably should fix the xibs as we go.

Reply:
@lucasderraugh Fixed.